src/main/java/king/Parser.java

```diff
@@ -123,8 +123,8 @@ public String parse(String phrase) throws KingException {
                 throw new KingException("Please follow the syntax: delete <item no.>", e);
```

Comment:
As a person taking the same project, I easily got the context and logic of your code. Perhaps you can improve readability by adding more abstractions or comments to guide other users into understanding your parse method.

Reply:
Thanks for your opinions!